src/client/refresh/vk/header/vk_mem_alloc.h

```diff
@@ -1,17087 +0,0 @@
-//
```

Comment:
Removing this file should be its own separate commit so it doesn't clutter up this one

Reply:
thank you, fixed in 6f3c223